src/components/transactions/AssetInput.tsx

```diff
@@ -146,6 +147,18 @@ export const AssetInput = <T extends Asset = Asset>({
 
           {!onSelect || assets.length === 1 ? (
             <Box sx={{ display: 'inline-flex', alignItems: 'center' }}>
+              {value !== '' && (
+                <Button
+                  size="small"
+                  sx={{ minWidth: 0, ml: '7px', p: 0, left: 8 }}
+                  onClick={() => {
+                    onChange && onChange('');
+                  }}
+                  disabled={disabled || isMaxSelected}
+                >
+                  <CancelIcon sx={{ fontSize: 16, fill: 'silver' }} />
```

Comment:
We typically use hero icons. I think the `XCircleIcon` from `@heroicons/react/solid` is what you could use.

Reply:
Fixed - now using `XCircleIcon`